src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -625,6 +602,38 @@ case class JesBackend(actorSystem: ActorSystem)
     }
   }
 
+  private def preemtped(errorCode: Int, backendCall: BackendCall) = errorCode == 13 && backendCall.preemptible
+
+  private def handleFailure(backendCall: BackendCall, errorCode: Int, errorMessage: Option[String], events: Seq[ExecutionEventEntry]) = {
+    import cromwell.util.IntUtil._
+
+    val taskName = s"${backendCall.workflowDescriptor.id}:${backendCall.call.unqualifiedName}"
+    val attempt = backendCall.key.attempt
+
+    if (errorMessage contains "Operation canceled at") {
+      AbortedExecutionHandle.future
+    } else if(preemtped(errorCode, backendCall)) {
+      val preemptedMsg = s"Task $taskName was preempted for the ${attempt.toOrdinal} time."
+      val max = backendCall.maxPreemption
+
+      if (backendCall.key.attempt < max) {
+        val e = new PreemptedException(
```

Comment:
See comment above - I'm not opposed to changing it to a String - I just thought it would be nice to keep the information that it's been preempted (when this bubbles up) as a type rather than just an error message.